### PR TITLE
add workspace deletion tools and readme

### PR DESCRIPTION
**Pull request by mmorgantaylor**

### CODE REVIEW

File: scripts/workspace_cleanup/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Workspace cleanup scripts
+
+The scripts in this directory can be used to delete Terra workspaces in bulk. Proceed with caution!
+
+`get_workspace_list.py` gathers a list of workspaces in a given project, along with information about storage cost and owners. That should be manually curated before passing it to the script to delete.
```
Comment (schaluva) [inline scripts/workspace_cleanup/README.md:5]: "That" --> the output of the script should be manually created?

File: scripts/workspace_cleanup/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+`delete_workspaces.py` deletes a list of workspaces.
+
+These script use the Terra APIs, so they can only delete workspaces that the running user has access to. If your goal is to view as many workspaces as possible in the project, it's best to make sure the person running the script is a project owner on the Terra project. Note that any workspace covered by an Authorization Domain (AD) will not be able to be deleted (or have cost information returned) by these scripts unless the person running the script is in the workspace's AD.
```
Comment (schaluva) [inline scripts/workspace_cleanup/README.md:9]: These scripts**s** (plural)

**Review by schaluva — APPROVED** [review]

File: scripts/workspace_cleanup/README.md (hunk 1 above)
Comment (mmorgantaylor) [reply scripts/workspace_cleanup/README.md:5]: Ah yes let me clarify. Yes, the output of that script is a csv and the user should manually curate/whittle it down to only the workspaces they want to delete (and only the `project` and `workspace` columns) to pass to the delete_workspaces script.

File: scripts/workspace_cleanup/README.md (hunk 2 above)
Comment (mmorgantaylor) [reply scripts/workspace_cleanup/README.md:9]: thank you!

**Pull request merged**